omod/src/main/webapp/pages/markPatientDead.gsp

```diff
@@ -0,0 +1,155 @@
+<%
+    ui.includeJavascript("coreapps", "custom/markPatientDead.js")
+    ui.decorateWith("appui", "standardEmrPage", [title: ui.message("coreapps.markPatientDead.label")])
+
+    def htmlSafeId = { extension ->
+        "${extension.id.replace(".", "-")}-${extension.id.replace(".", "-")}-extension"
+    }
+
+    Calendar cal = Calendar.getInstance()
+    def maxAgeYear = cal.get(Calendar.YEAR)
+    def minAgeYear = maxAgeYear - patient.getAge()
+    def breadcrumbMiddle = breadcrumbOverride ?: '';
+%>
+<script type="text/javascript">
+
+    var breadcrumbs = [
+        {icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm'},
+        {
+            label: "${ ui.escapeJs(ui.message("coreapps.markPatientDead.label")) }",
+            link: '${ ui.urlBind("/" + contextPath + "coreapps/clinicianfacing/patient.page?patientId="+patientId, [ patientId: patient ] ) }'
+        }
+    ];
+
+
+    jq(function () {
+        jq('#deceased').change(function () {
+            if (this.checked) {
+                showContainer('#death-date-container');
+                showContainer('#cause-of-death-container');
+            } else {
+                hideContainer('#death-date-container');
+                hideContainer('#cause-of-death-container');
+            }
+        });
+
+        jq('#deceased').each(function () {
+            if (this.checked) {
+                showContainer('#death-date-container');
+                showContainer('#cause-of-death-container');
+            } else {
+                hideContainer('#death-date-container');
+                hideContainer('#cause-of-death-container');
+            }
+        });
+
+        jq('#mark-patient-dead').submit(function () {
+            jq(".field-error").html("");
+
+            var hasError = false;
+            // check if the deceased checkbox is selected
+            if (jq('#deceased').is(":checked")) {
+                // validate that the date of death and cause of death are not empty
+                if (jq('#death-date-display').val()=="") {
+                    jq("#death-date > .field-error").append("Please select the date of death").show();
+                    hasError = true;
+                }
+                if (jq('#cause-of-death').val()=="") {
+                    jq("#cause-of-death-container > .field-error").append("Please select the cause of death").show();
+                    hasError = true;
+                }
+            }
+            return !hasError;
+        });
+
+    });
+</script>
+<style type="text/css">
```

Comment:
Shouldn't we put this css in a file like this? https://github.com/openmrs/openmrs-module-coreapps/blob/master/omod/src/main/compass/sass/patientdashboard/patientDashboard.scss